src/flynt/api.py

```diff
@@ -22,33 +23,63 @@
 blacklist = {".tox", "venv", "site-packages", ".eggs"}
 
 
+@dataclasses.dataclass(frozen=True)
+class FstringifyResult:
```

Comment:
Could we make it a named tuple to still support unpacking? I find it convenient to split variables right after the call.

Reply:
I'd rather not.

There's no point in attempting to try to unpack the result of e.g. `_fstringify_file` anyway since it could be `None` (instead of the [null object](https://en.wikipedia.org/wiki/Null_object_pattern) that `default_result()` used to return) if conversion failed.

Reply:
Also, unpacking a tuple is very prone to this class of bug:

```python
code, n_changes, new_length, old_length = some_call()
```
(see the bug? 😁)

Reply:
yeah, someone can unpack them into bad names.